packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -4,6 +4,10 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-combobox-input-height: Specifies the height of the component's input.
+ * @prop --calcite-combobox-selected-icon-color: Specifies the color of the `calcite-combobox-item`'s selected indicator icon.
```

Comment:
Why is the var `--calcite-combobox-selected-icon-color` documented on the combobox when it isn't used on the combobox at all?

Reply:
This should be reverted.